js/testapps/vertexai-reranker/package.json

```
@@ -8,7 +8,9 @@
     "compile": "tsc",
     "build": "pnpm build:clean && pnpm compile",
     "build:clean": "rimraf ./lib",
-    "build:watch": "tsc --watch"
+    "build:watch": "tsc --watch",
+    "genkit:dev": "cross-env GENKIT_ENV=dev pnpm dev",
```

This probably needs to be applied globally in all testapps -- we no longer need to set GENKIT_ENV and rely on cross-env
```suggestion
    "genkit:dev": "genkit start -- pnpm dev",
```